### ISSUE TITLE
Larger self buff/debuffs

### ISSUE THREAD

**Issue by owopeachy** [rgd87/BigPlayerDebuffs#3]
Could it be possible to add the same feature to your personal buff/debuff tray?
These ones:
![image](https://user-images.githubusercontent.com/37337528/148694128-1d95b8ec-c554-47ca-893f-ada4fb637a54.png)

Thanks for your work!

**Comment by owopeachy** [comment]
oh, this is a duplicate, mb

**Comment by d87** [comment]
I've been thinking about it myself recently. Won't promise it in the immediate future, but i'll check it out

**Comment by cyril-vandyke** [comment]
@d87 Would you mind if I forked this repo to try and addon this feature? It's something I've wanted for quite a while too.

**Comment by d87** [comment]
Sure, but the reason I never did it is because for one there's multiple possible configurations for buffs on player, but the real reason is uhh I keep forgetting again what is it exactly, but they're not sorted by the game or something. So technically it'll be on a way different level. While the game techincally already allows to split your own buffs from the rest

**Comment by cyril-vandyke** [comment]
Good to know! I'll still take a crack at it and we'll see how it goes :)

**Comment by RaphaelDDL** [comment]
> Good to know! I'll still take a crack at it and we'll see how it goes :)

Let us know when you got something.. I came here exactly to suggest this rofl.

I'm dumb and more often than not, I lose count of Requiescat counters when I need to do mechs and/or use mit for TBs and I end up using 5 Holy Spirits sometimes, losing the Confiteor cast.  Since the buff is unsorted in the middle of the buff bar, sometimes I don't locate in time to actually see if I should or not cast again _(or even in time to make a decision, and the buff expires)_ rofl

I believe XIVAuras theoretically could do that (showing the count in a bigger way on UI), but creator didn't wrote any tutorials nor I find any about it.. I tried meddling with it but I couldn't for life make it work like the screenshots he posted as example

**Comment by d87** [comment]
@RaphaelDDL I can suggest you JobBars plugin, it can add an indicator for Requiescat stacks in the form of 5 dots. And in general is good for filling the gaps in default gauges

**Comment by cyril-vandyke** [comment]
@RaphaelDDL I worked on this for a bit and ran into a myriad of issues as well. It's something I'd still like to get back to eventually :)

I'd also suggest the JobBars plugin, it's got a ton of job-specific actions like that!

**Comment by RaphaelDDL** [comment]
Yeah, thank you both for the suggestion. JobBars is incredible to my scenario. 
I'd still would like to see my own Buffs in the list bigger, specially since they are not sorted as own buffs first. But yeah, JobBars is great.
I'll even open up a suggestion for it, it has a sound alert fr when full, when empty, full or empty, but it doesn't have for a specific count. On Requiescat case, having a sound cue when on 1 left would be even greater. But anyway I digress, the 5 counter like Aetherflow is magnificent.

Thank you once more for the suggestion :D

**Comment by cyril-vandyke** [comment]
@RaphaelDDL 
> specially since they are not sorted as own buffs first.
This was recently added as a default setting. I don't have FFXIV open in front of me, but from the HUD settings, select your buff bar and go to settings and I believe there's a setting called "Prioritize own buffs" that moves them to the front.
